cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -1359,6 +1364,37 @@ def test_deprecated_fields(self):
             ],
         )
 
+    def test_not_enough_color_options(self):
+        self.validator.adata.uns["suspension_type_colors"] = ["green"]
+        self.validator.validate_adata()
+        self.assertEqual(
+            self.validator.errors,
+            [
+                "ERROR: Annotated categorical field suspension_type must have 2 color options in uns[suspension_type_colors]. Found: ['green']"
+            ],
+        )
+
+    def test_too_many_color_options(self):
```

Comment:
Is it a requirement that there are exactly the same number of colors as there are categories? I think the requirement was having a number of colors _greater than or equal to_ the number of categories.